client/app/dbaas/logs/detail/options/addtool/logs-add-tool.controller.js

```diff
@@ -0,0 +1,68 @@
+class logsAddToolCtrl {
+    constructor ($state, $stateParams, $translate, ControllerHelper, ControllerModalHelper, LogsOfferConstant, LogsOfferService) {
+        this.$state = $state;
+        this.$translate = $translate;
+        this.serviceName = $stateParams.serviceName;
+        this.ControllerHelper = ControllerHelper;
+        this.ControllerModalHelper = ControllerModalHelper;
+        this.LogsOfferConstant = LogsOfferConstant;
+        this.LogsOfferService = LogsOfferService;
+        this._initLoaders();
+    }
+
+    $onInit () {
+        const addTxt = this.$translate.instant("add_tool_add");
+        this.text = this.text || `${addTxt} ${this.toolName}`;
```

Comment:
`${addTxt} ${this.toolName}` won't work in many languages, including french, because of gender. Sometimes it will be "Ajouter un ..." other times "Ajouter une ..."
The default could simply be "Ajouter" or "Add".

Reply:
Add is now hard-coded in english as the default text